modules/creating-instance-aws-load-balancer-controller.adoc

```diff
@@ -85,8 +86,9 @@ spec:
 <2> Specifies the deployment name.
 <3> Specifies the number of replicas of the deployment.
 
-. Create a `service` resource:
+. Create a YAML file that defines the `Service` resource:
 +
+.Example `service-albo.yaml` file
```

Comment:
```suggestion
.Example `service-albo.yaml` file:
```

Reply:
@opayne1 @xenolinux I don't think we need a colon here, right? This is a heading.